keep/api/bl/incidents_bl.py

```diff
@@ -57,7 +56,19 @@ def create_incident(self, incident_dto: IncidentDtoIn) -> IncidentDto:
             "Creating incident",
             extra={"incident_dto": incident_dto.dict(), "tenant_id": self.tenant_id},
         )
-        incident = create_incident_from_dto(self.tenant_id, incident_dto)
+        if isinstance(incident_dto, IncidentDto):
+            incident_dict = {
+                "user_summary": incident_dto.user_summary,
+                "generated_summary": incident_dto.description,
+                "user_generated_name": incident_dto.user_generated_name,
+                "ai_generated_name": incident_dto.dict().get("name"),
+                "assignee": incident_dto.assignee,
+                "is_predicted": False,  # its not a prediction, but an AI generation
+                "is_confirmed": True,  # confirmed by the user :)
+            }
+            incident = create_incident_from_dict(self.tenant_id, incident_dict)
+        else:
+            incident = create_incident_from_dict(self.tenant_id, incident_dto.dict())
```

Comment:
can you add comment on when an incident_dto will be  IncidentDto and when it'll be dict?